lib/options/sort-order.js

```diff
@@ -283,7 +283,44 @@ module.exports = {
             propertyName = null;
             // Look for includes:
             if (node.get(i).is('include')) {
-                propertyName = '$include';
+                // Divide `include` into `$extend`,
+                // mixins with specific name (e. g. `$include breakpoint`),
+                // and the rest — `$include`.
+                var includeType;
+                var mixinName;
+
+                // SASS and SCSS both supports `@extend` and `@include`,
+                // but SASS also supports `+mixin-name`
+                if (_this.syntax === 'sass' || _this.syntax === 'scss') {
+                    if (_this.syntax === 'sass' && node.get(i).get(0).content === '+') {
+                        includeType = 'include';
+                        mixinName = node.get(i).get(1).get(0).content;
+                    } else {
+
+                        includeType = node.get(i).get(0).get(0).content;
+
+                        if (includeType === 'include') {
+                            mixinName = node.get(i).get(2).get(0).content;
+                        }
+                    }
+                }
+
+                if (_this.syntax === 'less') {
+                    includeType = 'include';
+                    mixinName = node.get(i).get(0).get(0).content;
+                }
+
+                if (includeType === 'extend') {
```

Comment:
Will it be easier if I divide `include` and `extend` nodes in Gonzales? So you'll have two separate node types to work with?